docs/meetings/exeter-jun-2019.md

```diff
@@ -0,0 +1,254 @@
+# Exeter Meeting Notes
+
+June 24-28
+Present HO, DM, MS, OS, SB, BK
+
+- ZenHub - use instead of Trello, as it provides a powerful multi-project view
+  of GitHub. 
+
+- Reviewed and merged a bunch of outstanding PRs
+  - esp. DS pair for Protobuf-over-ZMQ WFS-to-UIS comms 
+
+- UI Design Session led by OS
+  - refined our ideas based on OS excellent Inkscape mock-ups.
+  - OS has now updated the design document and posted to the cylc-ui issue.
+
+- MR provided instructions for his Cytoscape-based graph view experiments,
+  which look very promising, and a PR to the UI project. Uses fossilized test
+  data at this stage, but fine for demos. 
+
+- UI: prioritize Tree View based on OS mock-ups. Dot-view is low priority.
+
+- UIS-WFS authentication/trust - similar to CLI, e.g. one-time or timed token.
+
+- need encryption between Hub and UIS (not the default in J-Hub).
+  - high priority once we have the end-to-end system
+
+- UIS to do all authorization if possible. It has to do some, e.g. for suite
+  start-up, so might as well do it all.
+  - so audit trail for all client interaction needed at the UIS
+
+- User feedback request on Discourse
+  - agreed we should do this
+  - not wide spectrum of use cases - can't promise to please everyone
+  - SB to write up for review before posting to Discourse.
+
+- We will rely on J-Hub to be present, even for single-user installations
+
+- UIS needs to abort (or prob just be useless) if not started/owned by the Hub.
+
+- we may eventually need a native e-app (e.g. electron) for those without the
+  right network routing.
+
+- JupyterLab is a different/better notebook interface. Has a nice user-movable
+  tiling tab feature that we should try to use. 
+
+- Most tree view info should be retrieved on request only (e.g. Job ID etc. in
+  the current tree view, not needed unless the final level is expanded in the
+  new UI.
+
+- "rose suite-run" - need plugin-based file-installation, e.g. via fcm keywords
```

Comment:
Not sure what this means? I guess you are talking about supporting 3rd party file install schemes? The current functionality in Rose is already modular and plugin friendly. With the packaging change, it should become very easy to install plugins.

Reply:
Yes. Comment noted.